docs/swagger_v2/activities.spec.yaml

```diff
@@ -0,0 +1,149 @@
+schemas:
+  Aex9TransferEvent:
+    description: AEx9 Transfer
+    properties:
+  NameClaimEvent:
+    description: Name claim
+    properties:
+      tx_hash:
+        description: The hash of the transaction (either a name claim or a contract call transaction)
+        type: string
+      tx:
+        description: The claim transaction (internal or not)
+        type: object
+  InternalContractCallEvent:
+    description: Contract call made inside of a contract call or contract create transaction
+    properties:
+      contract_tx_hash:
+        description: The contract creation transaction hash
+        type: string
+      contract_id:
+        description: The contract pubkey
+        type: string
+      call_tx_hash:
+        description: The contract call transaction hash
+        type: string
+      function:
+        description: The name of the function
+        type: string
+      internal_tx:
+        description: The transaction object
+        type: object
+      height:
+        description: The height in which the internal call happened
+        type: integer
+      micro_index:
+        description: The micro block index
+        type: integer
+      block_hash:
+        description: The micro block hash in which the transaction happened
+        type: string
+  InternalTransferEvent:
+    description: Internal transfer
+    properties:
+      amount:
+        description: The amount of AE tokens
+        type: integer
+      kind:
+        description: The kind of internal transfer
+        type: string
+        enum:
+          - accounts_extra_lima
+          - accounts_fortuna
+          - accounts_genesis
+          - accounts_lima
+          - accounts_minerva
+          - contracts_lima
+          - reward_dev
+          - reward_block
+          - fee_refund_oracle
+          - fee_lock_name
+          - fee_refund_name
+          - fee_spend_name
+          - reward_oracle
+          - fee_lock_name
+      ref_tx_hash:
+        description: The transaction hash of a transaction related to the internal transfer
+        type: string
+  Activity:
+    description: Name Transfer
```

Comment:
More generic description needed to contemplate the other event types.

Reply:
Done